lib/puppet/functions/jira/is_installed.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# @summary Check if JIRA is already installed
+Puppet::Functions.create_function(:'jira::is_installed') do
```

Comment:
is there a reason why you used a function here that you defer? Usually such agent-side checks are done by a fact.

Reply:
Edit: Ah because you need to pass a user?